src/librustc_driver/pretty.rs

```diff
@@ -526,29 +526,33 @@ fn gather_flowgraph_variants(sess: &Session) -> Vec<borrowck_dot::Variant> {
 
 #[derive(Clone, Debug)]
 pub enum UserIdentifiedItem {
-    ItemViaNode(ast::NodeId),
+    ItemViaNode(hir::HirId),
     ItemViaPath(Vec<String>),
 }
 
 impl FromStr for UserIdentifiedItem {
     type Err = ();
     fn from_str(s: &str) -> Result<UserIdentifiedItem, ()> {
         Ok(s.parse()
-            .map(ast::NodeId::from_u32)
+            .map(|n| {
```

Comment:
This doesn't work at all since you can't represent HirIds with an integer.

Reply:
I thought I could get away with `hir::ItemLocalId::from_u32(n)` :slightly_smiling_face: ; I'll revert that bit.